Guide/mail.markdown

```diff
@@ -196,4 +221,62 @@ instance BuildMail ConfirmationMail where
 
 ## Plain Text Emails
 
-TODO
+Every email should have a plain text version for people with reasonable mail clients. If you don't specify one and only set the HTML content via `html` (see above), then IHP automatically creates a plain text version from you by stripping away all HTML tags. This is suboptimal.
+
+The better option is to manually provide a useful plain text version of your emails:
+
+```haskell
+-- don't use, emails look off
+
+text ConfirmationMail { .. } = cs [plain|
```

Comment:
Actually it's best now to use `import NeatInterpolation` for doing this. This package automatically deals with the indentation. It exports a `[trimming||]` quasiquoter. https://hackage.haskell.org/package/neat-interpolation-0.5.1.2/docs/NeatInterpolation.html

It's already used in many places in IHP itself, [e.g. the code generators](https://github.com/digitallyinduced/ihp/blob/master/IHP/IDE/CodeGen/ViewGenerator.hs#L101).

Reply:
Yes of course Haskell has a proper solution for this :facepalm:  

Docs changed accordingly, thanks!